src/rpcwallet.cpp

```diff
@@ -2481,16 +2481,31 @@ UniValue listspentzerocoins(const UniValue& params, bool fHelp)
 
 UniValue mintzerocoin(const UniValue& params, bool fHelp)
 {
-    if (fHelp || params.size() != 1)
+    if (fHelp || params.size() < 1 || params.size() > 2)
         throw runtime_error(
-            "mintzerocoin <amount>\n"
-            "Usage: Enter an amount of Piv to convert to zPiv"
+            "mintzerocoin <amount> [UTXOs]\n"
```

Comment:
change to:
```
"mintzerocoin amount ( [UTXOs] )\n"
```

required arguments have no braces/brackets around them, optional arguments are enclosed in parentheses with a space on either side of the optional argument.